<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemView</class>
 <widget class="QWidget" name="ItemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>198</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item>
    <widget class="TitleWidget" name="title"/>
   </item>
   <item>
    <widget class="SearchWidget" name="searchWidget"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="treeLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="TreeView" name="treeView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="ListView" name="listView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header>widgets/titlewidget.h</header>
  </customwidget>
  <customwidget>
   <class>PaddedSqueezedTextLabel</class>
   <extends>QLabel</extends>
   <header>support/squeezedtextlabel.h</header>
  </customwidget>
  <customwidget>
   <class>SearchWidget</class>
   <extends>QLineEdit</extends>
   <header>widgets/searchwidget.h</header>
  </customwidget>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>widgets/treeview.h</header>
  </customwidget>
  <customwidget>
   <class>ListView</class>
   <extends>QListView</extends>
   <header>widgets/listview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
